Solution for problem 11

juergenbr opened the pull request:
**Description**
Solution for problem 11.
Random grid is generated and then checked for largest sum, size can be set via global variable.
Output contains full grid, largest calculated sum and position in the grid.

**Issues**
Resolves #89

yafiwebdev reviewed (commented):
Hey, thank you for providing this solution, especially the visual output!
But I'm really sorry to say that this doesn't seem to be a solution for the project euler problem No. 11, you're outputting max sum and not product. (Let me know if I'm mistaken)

Furthermore, although it is brilliant to show visual output, but the main issue is to solve the problem, so it would be better to define the fillGrid() at the very end of the file and move the more important functions to the top for people to see them more easily.

Again, thank you for providing this nice solution!

juergenbr commented:
@yafiwebdev Hi, thanks for the feedback! I'm sorry I totally misread the problem description. Code is now fixed to calculate the product.
I moved the fillGrid method down and kept the calculation methods at the top, hope this makes it more clear. Let me know what you think!

yafiwebdev reviewed (approved):
Merging
